test/js-api/global/constructor.any.js

```diff
@@ -90,8 +90,9 @@ test(() => {
 test(() => {
   const argument = { "value": "i64" };
   const global = new WebAssembly.Global(argument);
-  assert_throws(new TypeError(), () => global.value);
-  assert_throws(new TypeError(), () => global.valueOf());
+
+  assert_equals(global.value, 0n, "initial value using ToJSValue");
```

Comment:
How widely supported is this syntax? (JSC?) Consider using `BigInt("0")` or something else that doesn't cause the whole test to be a syntax error if needed.